docs/topic2.md

```diff
@@ -1,3 +1,115 @@
-# Topic 2 - CSS & HTML
+# Topic 2 - CSS
+
+[-> To index](../README.md#title)
+
+
+![](https://thumbs.gfycat.com/FirstWarpedInganue-size_restricted.gif)
+
+## Documentation
+
+- https://developer.mozilla.org/en-US/docs/Learn/CSS
+- [Selectors](https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Selectors)
+- [CSS3 selectors sheet](https://www.w3.org/TR/selectors-3/)
+- [Specificity](https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Cascade_and_inheritance#Specificity)
+- [Box-model](https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_boxes/Box_model_recap)
+- Learn about the Box Model (how the browser calculates boxes size): http://www.w3.org/TR/CSS21/box.html
+
+# Exercises
+ Use the next html file to do the exercies
+  ```
+  <!doctype html>
```

Comment:
The markup could be moved into a folder like assets and then make link to the place